AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -3148,6 +3148,7 @@ smashbros.com##.cookie-notes
 aptoide.com##.cookie-notification-holder
 calik.com,telekom.hu##.cookie-policy-container
 zebramo.com,modacruz.com##.cookie-policy-sticky
+bonus.com.tr###cookie-policy-sticky
```

Comment:
After hiding an element, a white stripe remains

I would add these rules:
`bonus.com.tr#%#//scriptlet('set-cookie', 'cookie-agreement', '1')`
`bonus.com.tr#@#.cookie-content` - to cookies_whitelist

Reply:
`bonus.com.tr###cookie-policy-sticky` it's ok for cb platforms. `bonus.com.tr#@#.cookie-content` I don't think this is necessary.

@gork7777 `cookies_specific.txt` add `bonus.com.tr#%#//scriptlet('set-cookie', 'cookie-agreement', '1')` under `JS rules`.
If you can't do that please let us know.

Reply:
If you hide it for content blocker, padding will not be removed, and message(leftover) cannot be closed by user.
![image](https://user-images.githubusercontent.com/8361299/110128275-4a1c4100-7dcf-11eb-9583-7d23972a0342.png)
Better just scriptlet or CSS.

Reply:
@Alex-302 isn't it removed after scrolling? Then I guess scriptlet / css are better here with exception, but without the basic rule from the pull request.

Reply:
After scrolling:
![image](https://user-images.githubusercontent.com/8361299/110129428-9f0c8700-7dd0-11eb-8aef-8e0259c24d55.png)


Just scriptlet/CSS is required. Without exclusion, close button is visible.

Reply:
Ok, thank you.

Reply:
Fixed and how did you find the 'cookie-agreement'? I don't know these scriptlet codes very well

Reply:
It's not me who found it. But you can see it after accepting cookies in devtools -> application -> cookies or look at their js code.

<details><summary>For instance:</summary>

![image](https://user-images.githubusercontent.com/33488229/110130944-a9338300-7dda-11eb-9195-b784c20df748.png)

</details><br/>

Reply:
Okay thanks for information!

Reply:
You are welcome.